courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
<p>First sentence, first paragraph - change "&nbsp;creare"  to "creatre.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>